examples/ads.amp.html

```diff
@@ -1085,6 +1086,14 @@ <h2 id="sovrn">SOVRN</h2>
       data-z="393900"><!-- this ID is only whitelisted for localhost:8000 and http://amphtml-nightly.herokuapp.com-->
   </amp-ad>
 
+  <h2 id="spotx">SpotX</h2>
+  <amp-ad width="300" height="250"
+          type="spotx"
+          data-spotx_channel_id="85394"
```

Comment:
We are hoping to match the formatting here to that used in another of our popular integrations making it easier for publishers to adopt this. In that integration, options are passed to our script using the `data-spotx_<option>` format.